### ISSUE TITLE
Incorrect highlighting with variadic arguments and type hints.

### ISSUE THREAD

**Issue by Gert-dev** [atom/language-php#117]
Hello

It seems variadic function/method parameters with type hints are incorrectly higlighted (PHP >= 5.6):

```php
public function tesarsdhienarsdhinearsd(SomeBaseClass $param1, SomeBaseClass ...$param2)
{

}
```

Here `SomeBaseClass` is not highlighted as class. (I notice now that GitHub's highlighting conveniently has exactly the same problem.)

Thanks in advance

**Comment by zr9** [comment]
Added a fix(pull request) for that issue, but not know how attach commit here

**Comment by 50Wliu** [comment]
@zr9 Type "fixes #issue" in either the commit or the description of your PR.

For example:
`Fixes #117`

**Comment by zr9** [comment]
Oh thanks for advice, i'm use in future updates

**Comment by zr9** [comment]
Updated, added specs and support for **meta.function.argument.no-default.php** scope

**Comment by 50Wliu** [comment]
Fixed by #120.
